frontend/src/scenes/actions/ActionsTable.tsx

```diff
@@ -46,8 +46,39 @@ export function ActionsTable(): JSX.Element {
     const { loadActions } = useActions(actionsModel)
     const [searchTerm, setSearchTerm] = useState('')
     const [filterByMe, setFilterByMe] = useState(false)
+    const [dupVisible, setDupVisible] = useState(false)
+    const [dupName, setDupName] = useState('')
+    const [dupAction, setDupAction] = useState<ActionType>()
     const { user, hasAvailableFeature } = useValues(userLogic)
 
+    const duplicateAction = async (action: ActionType | undefined): Promise<void> => {
+        if (action) {
```

Comment:
If you pass the action directly to this function, there is no need to handle the undefined case. Thus you should be able to remove this check and the associated typing in the function declaration.